version/release_test.go

```diff
@@ -1,62 +1,116 @@
 package version
 
 import (
+	"fmt"
 	"strings"
 	"testing"
 )
 
-func TestInfoManaged(t *testing.T) {
+func setupDataForManaged() {
 	ProjectName = "pmm-managed"
 	Version = "2.1.2"
 	PMMVersion = "2.1.2"
 	Timestamp = "1545226908"
 	FullCommit = "6559a94ab33831deeda04193f74413b735edb1a1"
 	Branch = "master"
+}
+
+func setupDataForExporter() {
+	ProjectName = "external_exporter"
+	Version = "0.8.5"
+	PMMVersion = "2.1.2"
+	Timestamp = "1545226909"
+	FullCommit = "6559a94ab33831deeda04193f74413b735edb1a2"
+	Branch = "master"
+}
+
+func TestShortInfoManaged(t *testing.T) {
+	setupDataForManaged()
 
-	expected := "pmm-managed v2.1.2"
+	expected := fmt.Sprintf("%s v%s", ProjectName, PMMVersion)
 	actual := ShortInfo()
 	if expected != actual {
 		t.Errorf("expected: %q\nactual: %q", expected, actual)
 	}
+}
+
+func TestFullInfoPlainManaged(t *testing.T) {
+	setupDataForManaged()
 
-	expected = strings.Join([]string{
-		"ProjectName: pmm-managed",
-		"Version: 2.1.2",
-		"PMMVersion: 2.1.2",
-		"Timestamp: 2018-12-19 13:41:48 (UTC)",
-		"FullCommit: 6559a94ab33831deeda04193f74413b735edb1a1",
-		"Branch: master",
+	expected := strings.Join([]string{
+		fmt.Sprintf("ProjectName: %s", ProjectName),
```

Comment:
TBH, I don't like that we use almost the same code for expected result as for the actual code. I would prefer to use strings if possible.

Reply:
Only data is the same, need to check that passed data to func and returned are equal. The rest (formatting, Join) is just helpers to construct the expected result. To my mind it improves code maintainability - no need to look through the whole sis and find where to change the strings